src/toil_scripts/lib/test/test_urls.py

```diff
@@ -22,6 +22,14 @@ def test_upload_and_download_with_encryption(tmpdir):
     from toil_scripts.lib.urls import s3am_upload
     from toil_scripts.lib.urls import download_url
     from boto.s3.connection import S3Connection, Bucket, Key
+    # Check if exists and delete if it does
```

Comment:
I guess I'm being paid to be the teams broken record. There could be multiple concurrent builds. This is racy and unhygienic, i.e. leaves crap around. Randomize the output location and do clean-up with try: finally:.